### PR TITLE
Updates default config.yaml to use Salt 3003.3

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

**Comment by eemperor** [comment]
go codebuild go

File: src/watchmaker/static/config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-watchmaker_version: ">= 0.21.9"
+watchmaker_version: ">= 0.23.2"
```
Comment (lorengordon) [inline src/watchmaker/static/config.yaml:1]: i don't think we need to change this... this is for when we modify config in a way that the parser of an earlier version won't recognize the format/type/syntax... in this case, an earlier version of watchmaker could use this config just fine.

Comment (eemperor) [reply src/watchmaker/static/config.yaml:1]: Yeah.  I was questioning whether it was needed but wasn't sure.  I'll undo the change.

**Review by lorengordon — APPROVED** [review]
:shipit:

**Pull request merged**